README.md

```diff
@@ -4,15 +4,27 @@ Code style used by AlpineLab in all our projects.
 
 ## Usage
 
```

Comment:
Maybe we could keep sub-title
`### Gem integration`

Reply:
Your call, I have no specific opinion about this at the time.